x-pack/plugins/cases/public/components/markdown_editor/eui_form.tsx

```diff
@@ -44,11 +51,16 @@ export const MarkdownEditorForm = React.memo(
         bottomRightContent,
         caseTitle,
         caseTags,
+        draftCommentStorageKey,
         disabledUiPlugins,
+        initialValue,
       },
       ref
     ) => {
       const { isInvalid, errorMessage } = getFieldValidityAndErrorMessage(field);
+      const [showVersionConflictWarning, setShowVersionConflictWarning] = useState<boolean>(false);
+      const storage = useMemo(() => new Storage(window.sessionStorage), []);
```

Comment:
Let's use [`useSessionStorage`](https://github.com/streamich/react-use/blob/master/docs/useSessionStorage.md) from the [`react-use`](https://github.com/streamich/react-use) library. The hook does not export a `remove` function so we need to set the value to an empty string.